lib/vanagon/platform/windows.rb

```diff
@@ -50,22 +53,73 @@ def generate_packaging_artifacts(workdir, name, binding)
       # Method to generate the files required to build an MSI  package for the project
       #
       # @param workdir [String] working directory to stage the evaluated templates in
-      # @param name [String] name of the project
+      # @param project [Vanagon::Project] project to build a windows package of
+      # @param binding [Binding] binding to use in evaluating the packaging templates
+      public
```

Comment:
At this point this discussion is moot. As Mike noted later in this PR, private public is out of the scope of this work. It makes sense to do at some point, but that work should include more than just the work in this PR. I'm just going to take it out, changes to come soon.